scraper/tfs/__init__.py

```diff
@@ -0,0 +1,187 @@
+#! /usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+import logging
+import os
+
+from vsts.vss_connection import VssConnection
+from msrest.authentication import BasicAuthentication
```

Comment:
Where does `msrest` come from? Is that included with the `vsts` package? If not, I think it might be missing from the requirements file.

Reply:
Thanks, it's a dependency from vsts. I added it to production.txt. I don't actually care what version as that's driven by vsts but pegged it to 0.6.4 as that's what is downloaded by vsts 0.1.25.

Reply:
If installing vsts will pull it in, then we can probably leave it out.